### DISCUSSION TITLE
ERROR: Could not build wheels for pycocotools

### DISCUSSION THREAD

**Discussion by MABatin** [open-mmlab/mmdetection#7908] [Q&A]
I am getting following error when trying to install mmdet on windows. it's the pycocotools package that seems to be making the problem. Any solution or workaround for this would be highly appreciated
![Capture](https://user-images.githubusercontent.com/56408528/166725888-8ec0abbd-92c5-4aca-a274-ad34965676d6.PNG)
.

**Comment by ansharora7** [comment]
Take a look at this [https://github.com/cocodataset/cocoapi/issues/169](url)
